src/url.js

```diff
@@ -67,7 +67,7 @@ export function parseUrl(url) {
  * @param {string} paramString
  * @return {string}
  */
-function appendParamStringToUrl(url, paramString) {
+export function appendParamStringToUrl(url, paramString) {
```

Comment:
What are our XSS risks here if we allow arbitrary URL injection without encoding?